src/lib.rs

```diff
@@ -378,7 +378,7 @@ impl Surface {
 ///
 /// # Data representation
 ///
-/// The format of the buffer is as follows. There is one `u32` in the buffer for each pixel in
+/// The format of the buffer is `BGRX8888`, which is as follows. There is one `u32` in the buffer for each pixel in
```

Comment:
```suggestion
/// The format of the buffer is `BGR0`, which is as follows. There is one `u32` in the buffer for each pixel in
```
I'm not versed in these kind of terms, is there some precedence for naming it like you did?

Reply:
It's based on how Wayland formats are described, but I like your description better